moran_data/moran/main.py

```diff
@@ -0,0 +1,152 @@
+import csv
+
+import axelrod as axl
+import numpy as np
+import pandas as pd
+
+import opt_mo
+
+
+def theoretic_fixation(N, payoff_matrix, K=1):
+    """
+    Calculate x_i assuming a non dynamic player
+    """
+    f_ones = np.array(
+        [
+            (payoff_matrix[0, 0] * (K - 1) + payoff_matrix[0, 1] * (N - K))
+            for K in range(1, N)
+        ]
+    )
+    f_twos = np.array(
+        [
+            (payoff_matrix[1, 0] * K + payoff_matrix[1, 1] * (N - K - 1))
+            for K in range(1, N)
+        ]
+    )
+    gammas = f_twos / f_ones
+    return (1 + np.sum(np.cumprod(gammas[: K - 1]))) / (
+        1 + np.sum(np.cumprod(gammas))
+    )
+
+
+def obtain_payoff_matrix(opponent, K, N=4):
+    """
+    Obtain the payoff matrix for K best response players in a population of a
+    total of N individuals.
+    """
+    best_ev_response, _, _ = opt_mo.get_evolutionary_best_response(
+        [opponent], opt_mo.get_memory_one_best_response, K=K
```

Comment:
This needs to be `[opponent] * (N - K)`

```suggestion
        [opponent] * (N - K), opt_mo.get_memory_one_best_response, K=K
```